src/main/java/uk/gov/gchq/koryphe/impl/binaryoperator/StringDeduplicateConcat.java

```diff
@@ -41,10 +41,14 @@
     protected String _apply(final String a, final String b) {
         final Set<String> set = new LinkedHashSet<>();
 
-        set.addAll(Arrays.asList(a.split(separator)));
-        set.addAll(Arrays.asList(b.split(separator)));
+        if (a != null) {
```

Comment:
a and b cannot be null as null checks are performed in KorypheBinaryOperator.
You should update you unit tests to call the 'apply' method instead of '_apply' if you want to test the null handling.